views/w3c/groups.js

```diff
@@ -64,18 +64,11 @@ module.exports = ({ groups }) => html`
       <h1>W3C Working Groups and Community Groups supported by ReSpec</h1>
       <p>
         List of possible values for
-        <a href="/docs/#group"><code>respecConfig.group</code></a>.
-        Send a Pull Request to <a href=${PR_URL}>add missing groups.</a>
```

Comment:
No need to send PR now. We've everything!

Reply:
nice!